src/Client/Raylib/Graphics/Graphics.cpp

```diff
@@ -0,0 +1,710 @@
+    {
+        ::Vector2 pos = {position.X(), position.Y()};
+        DrawTextureV(_texture, pos, {tint.R(), tint.G(), tint.B(), tint.A()});
+    }
+
+    void
+    Sprite::drawEx(Vector2 position, float rotation, float scale, Color tint)
+    {
+        ::Vector2 pos = {position.X(), position.Y()};
+        DrawTextureEx(
+            _texture,
+            pos,
+            rotation,
+            scale,
+            {tint.R(), tint.G(), tint.B(), tint.A()});
+    }
+
+    void Sprite::drawRec(Rectangle source, Vector2 position, Color tint)
+    {
+        ::Rectangle src =
+            {source.X(), source.Y(), source.Width(), source.Height()};
+        ::Vector2 pos = {position.X(), position.Y()};
+        DrawTextureRec(
+            _texture,
+            src,
+            pos,
+            {tint.R(), tint.G(), tint.B(), tint.A()});
+    }
+
+    void Sprite::drawPro(
+        Rectangle source,
+        Rectangle dest,
+        Vector2 origin,
+        float rotation,
+        Color tint)
+    {
+        DrawTexturePro(
+            _texture,
+            {source.X(), source.Y(), source.Width(), source.Height()},
+            {dest.X(), dest.Y(), dest.Width(), dest.Height()},
+            {origin.X(), origin.Y()},
+            rotation,
+            {tint.R(), tint.G(), tint.B(), tint.A()});
+    }
+
+    // Color/pixel related functions
+    Color fade(Color color, float alpha)
+    {
+        ::Color c = {color.R(), color.G(), color.B(), color.A()};
+        ::Color f = Fade(c, alpha);
+        return Color {f.r, f.g, f.b, f.a};
+    }
+
+    int colorToInt(Color color)
+    {
+        ::Color c = {color.R(), color.G(), color.B(), color.A()};
+        return ColorToInt(c);
+    }
+
+    Vector4 colorNormalize(Color color)
+    {
+        ::Color c   = {color.R(), color.G(), color.B(), color.A()};
+        ::Vector4 v = ColorNormalize(c);
+        return Vector4 {v.x, v.y, v.z, v.w};
+    }
+
+    Color colorFromNormalized(Vector4 normalized)
+    {
+        ::Vector4 v =
+            {normalized.X(), normalized.Y(), normalized.Z(), normalized.W()};
+        ::Color c = ColorFromNormalized(v);
+        return Color {c.r, c.g, c.b, c.a};
+    }
+
+    Color getColor(unsigned int hexValue)
+    {
+        ::Color c = GetColor(hexValue);
+        return Color {c.r, c.g, c.b, c.a};
+    }
+
+    // Text functions
+    void
+    drawText(std::string text, int posX, int posY, int fontSize, Color color)
+    {
+        DrawText(
+            text.c_str(),
+            posX,
+            posY,
+            fontSize,
+            {color.R(), color.G(), color.B(), color.A()});
+    }
+
+    void drawFPS(int posX, int posY)
+    {
+        DrawFPS(posX, posY);
+    }
+
+    int measureText(const std::string text, int fontSize)
+    {
+        return MeasureText(text.c_str(), fontSize);
+    }
+
+    Text::Text(std::string text, Vector2 position, float fontSize, Color color)
+        : _text(std::move(text)),
```

Comment:
pourquoi move text ? met le en const et copy le
```suggestion
    Text::Text(const std::string &text, Vector2 position, float fontSize, Color color)
        : _text(text),
```

Reply:
Je suis juste trop moderne mec
 Graphics.cpp(652,16): error GDF8306C9: pass by value and use std::move [modernize-pass-by-value]

Reply:
chelou